frontend/src/pages/notebookController/SpawnerPage.tsx

```diff
@@ -300,7 +300,9 @@ const SpawnerPage: React.FC<SpawnerPageProps> = React.memo(({ setStartModalShown
       const pvcName = generatePvcNameFromUsername(username);
       const pvc = await getPvc(pvcName);
       if (!pvc) {
-        await createPvc(pvcName, '20Gi');
+        const defaultPvcSize = '20Gi';
```

Comment:
Instead of defining the default PVC size here every time, could you move it to const.ts like `DEFAULT_PVC_SIZE` and import it to this file?